### DISCUSSION TITLE
FreeCAD to I-Simpa

### DISCUSSION THREAD

**Discussion by harerton** [Ifsttar/I-Simpa#285] [Q&A]
Hi,

Is there a tutorial on how to create a room using FreeCAD and import it to I-Simpa?

I would like to create a detailed auditorium model.

Thanks!

**Comment by nicolas-f** [comment]
Hi,

There is no such tutorial but I have some useful hints:

 - The most compatible method to export the model is to use the Mesh Workbench and to convert the model into a mesh
 - Then export this mesh in stl format

The last version of I-Simpa can import in millimeter using the dropdown on import (it is the default unit of FreeCad)
